eos-boot-upgrade.service

```diff
@@ -0,0 +1,19 @@
+[Unit]
+Description=Updater for GRUB/SHIM
+DefaultDependencies=no
+Conflicts=shutdown.target
```

Comment:
OOI what happens if this unit is running when the user shuts down the system? Will this be killed? (I appreciate it is extremely unlikely.)